src/components/ChatWelcome.tsx

```diff
@@ -28,18 +37,24 @@ export const ChatWelcome = ({
         >
           <Logo size={72} bgColor={"transparent"} />
         </div>
-        <div className={cx(`text-center mt-16 ${brandText}`)}>
-          <h4 className={cx(`${brandText}`)}>
+        <div className={cx(`text-center mt-16`)}>
+          <h4 className={cx(`${brandText} ${brandWeight}`)}>
             {import.meta.env.VITE_ASSISTANT_NAME}
           </h4>
-          <p className="my-4 text-small !font-normal">{subHeader}</p>
+          <p className={cx(`my-4 text-small !font-normal ${brandSansSerif}`)}>
+            {subHeader}
+          </p>
         </div>
       </div>
       <div className="flex flex-col gap-8 items-start">
-        <h5 className={cx(`text-large font-bold ${brandText}`)}>
+        <h5 className={cx(`text-large ${brandText} ${brandWeight}`)}>
```

Comment:
Med cx är det smidigt att bara skicka in variabeln som den är, den sköter validering av undefined osv